pyanaconda/modules/network/network.py

```diff
@@ -686,24 +685,23 @@ def get_dracut_arguments(self, iface, target_ip, hostname, ibft):
                     if active_connection:
                         target_connections = [active_connection.get_connection()]
 
-        if target_connections:
-            if len(target_connections) > 1:
-                log.debug("Get dracut arguments: "
-                          "multiple connections found for traget %s: %s, taking the first one",
-                          [con.get_uuid() for con in target_connections], target_ip)
-            connection = target_connections[0]
-        else:
-            log.error("Get dracut arguments: can't find connection for target %s", target_ip)
-            return dracut_args
-
-        dracut_args = list(get_dracut_arguments_from_connection(
-            self.nm_client,
-            connection,
-            iface,
-            target_ip,
-            hostname,
-            ibft
-        ))
+            if target_connections:
+                if len(target_connections) > 1:
+                    log.debug("Get dracut arguments: "
+                              "multiple connections found for traget %s: %s, taking the first one",
```

Comment:
I know this is merged but we have a typo here which should be fixed :).

 `traget` -> `target`

I know I'm such a tragéd ;).

Reply:
https://github.com/rhinstaller/anaconda/pull/3760